backend/platal/auth/factories.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+import datetime
+import django.utils.text
+import django.utils.timezone
+import factory
+import factory.django
+
+from platal.auth import models
+
+
+class AccountFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = models.Account
+
+    firstname = factory.Faker('first_name')
+    lastname = factory.Faker('last_name')
+    full_name = factory.LazyAttribute(lambda o: '{} {}'.format(o.firstname, o.lastname))
```

Comment:
I'm sure this can be written ``LazyAttribute("{0.firstname} {0.lastname}".format)`` ;)